docs/micropython.md

```diff
@@ -36,10 +37,42 @@ See [the MicroPython differences page](http://docs.micropython.org/en/latest/gen
 
 You should modify the first line of your scripts to replace "python3" with "micropython":
 
-```
+```python
 #!/usr/bin/env micropython
 ```
 
 ### Running from the command line
 
-If you previously would have typed `python3 foo.py`, you should now type `micropython foo.py`.
\ No newline at end of file
+If you previously would have typed `python3 foo.py`, you should now type `micropython foo.py`.
+
+If you are running programs via an SSH shell, use the following command line to inform Brickman that your
+program needs control of the EV3:
+
+```shell
+brickrun -- ./program.py
+```
+
+### Displaying text on the EV3 LCD screen
+
+The EV3 LCD console supports ANSI codes to position the cursor, clear the screen, show text with different fonts,
+and use reverse (white text on black background, versus the default black text on white background). See the file
+`utils/display_fonts.py` to see the fonts supported with sample calls to the `ev3dev2.Display()` class.
+
+### Building and installing the latest EV3DEV2 module on your EV3
+
+In an SSH Terminal window with an EV3 with Internet access, run the following commands:
+(recall that the `sudo` password is `maker`)
+
+```shell
+git clone https://github.com/ev3dev/ev3dev-lang-python.git
+cd ev3dev-lang-python
+sudo make
+```
+
+To update the module, use the following commands:
+
+```shell
+cd ev3dev-lang-python
+git pull
+sudo make
```

Comment:
same here

Reply:
Good. Done. But committed in the Console() PR #649.